### PR TITLE
P/stieg/readme

### PR DESCRIPTION
**Pull request by stieg**
README redone.  I have yet to apply some of the original feedback but I will.  Have a look and LMK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,150 @@
...
+### Toolchain Setup
+Do the following to setup the MK1 toolchain
+* Download [the official MK1
+  toolchain](https://s3-us-west-2.amazonaws.com/asl-firmware/vagrant_setup/gnuarm-4.0.2.tgz)
+* Extract the tarball to a directory of your choice
+* Add the 'bin' directory from within the 'gnuarm-4.0.2' directory to
+  your system PATH.
 
-Once the firmware completes flashing, RaceCapture/Pro will reset and run normally with the default configuration.
+### Compiling LUA for MK1
```
Comment (brentpicasso) [inline README.md]: launching do_release.sh builds everything in one shot. worth mentioning

File: README.md [hunk 2]
```diff
@@ -1,59 +1,150 @@
...
+platforms are hardware based but the testing platform is a pseudo
+platform that we use for mock operations to test our firmware.  Info
+on each platform and how to use/compile for it can be found below.
 
-#flasher RaceCapturePro-X.X.X.elf
+## MK1
+MK1 was the original RaceCapturePro unit.  Based on the AT91SAM7s
+chipset, it provided a solid platform for our initial release.  It
+allowd for up to a 100Hz channel sampling rate and up to 10Hz GPS.  It
+also supported CAN & OBD2, various I/O, an SD card, and a remote GPS
```
Comment (brentpicasso) [inline README.md]: supported CAN & OBD-II via the external CANx module

File: README.md [hunk 3]
```diff
@@ -1,59 +1,150 @@
-RaceCapture/Pro Firmware
-========================
+RaceCapturePro Firmware
```
Comment (brentpicasso) [inline README.md]: Make all name RaceCapture/Pro consistent throughout doc

**Comment by stieg** [comment]
I applied your feedback @brentpicasso.  Have a look and if you like it plz merge.

**Pull request merged**